src/elements/hx-beacon/index.js

```diff
@@ -0,0 +1,53 @@
+import { HXElement } from '../../interfaces/HXElement/index.js';
+
+import shadowMarkup from './_shadow.html';
+import shadowStyles from '../../scss/components/beacon/index.scss';
```

Comment:
`shadowStyles` should be self-encapsulated in `./_shadow.scss`.  If the styles in `../../scss/components/beacon/index.scss` are not needed in the Light DOM, please move them here.

Reply:
Changes have been done, please review